### PR TITLE
GitOps annex and references

### PR DESCRIPTION
**Pull request by taylor**
Adding some information on GitOps since it is references in the whitepaper

### CODE REVIEW

**Review by agentpoyo — APPROVED** [review]
LGTM

**Review by agentpoyo — APPROVED** [review]
Additions look good.

File: doc/whitepaper/Accelerating_Cloud_Native_in_Telco.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Accelerating Cloud Native in Telco
 **_Challenges of Cloud Native Telco Transformation today and how to overcome them - A CSP perspective_**
 
-v1.0 - November 3, 2023
+v1.0 - December 3, 2023
```
Comment (lixuna) [inline doc/whitepaper/Accelerating_Cloud_Native_in_Telco.md]:
```suggestion
v1.0 - December 4, 2023
```

File: doc/whitepaper/Accelerating_Cloud_Native_in_Telco.md [hunk 2]
```diff
@@ -322,12 +322,31 @@ CNFs need to adopt these principles as well. CNFs need to be:
...
 
+## ANNEX 2
 
-## ACKNOWLEDGEMENTS
+### GitOps for cloud native applications and infrastructure
 
+GitOps is not a single product, plugin, or platform. While the practices and patterns in GitOps existed before Cloud Native (and the term GitOps), they happen to be a great match for cloud native applications and infrastructure alike.
 
-Special acknowledgments go to the following Communication Service Providers, who have contributed to this whitepaper: 
+Here are some principles for GitOps (as defined by the Open GitOps community):
```
Comment (lixuna) [inline doc/whitepaper/Accelerating_Cloud_Native_in_Telco.md]:
```suggestion
Here are some principles for GitOps (as defined by the OpenGitOps community):
```

File: doc/whitepaper/Accelerating_Cloud_Native_in_Telco.md [hunk 3]
```diff
@@ -322,12 +322,31 @@ CNFs need to adopt these principles as well. CNFs need to be:
...
+
+- Declarative - A system managed by GitOps must have its desired state expressed declaratively.
+- Versioned and Immutable - The desired state is stored in a way that enforces immutability, versioning and retains a complete version history.
+- Pulled Automatically - Software agents automatically pull the desired state declarations from the source.
+- Continuously Reconciled - Software agents continuously observe actual system state and attempt to apply the desired state.
+
+GitOps generally has the following components:
+- A version control repository as the single source of truth for infrastructure and application definitions.
+- Merge requests (or pull requests (PRs)) as the change mechanism for all infrastructure updates.
+- A git (or version-controlled repository) workflow supporting automation of  infrastructure and application updates when new code is merged with continuous integration and continuous delivery (CI/CD)
```
Comment (lixuna) [inline doc/whitepaper/Accelerating_Cloud_Native_in_Telco.md]:
```suggestion
- A git (or version-controlled repository) workflow supporting automation of infrastructure and application updates when new code is merged with continuous integration and continuous delivery (CI/CD).
```

File: doc/whitepaper/Accelerating_Cloud_Native_in_Telco.md [hunk 4]
```diff
@@ -322,12 +322,31 @@ CNFs need to adopt these principles as well. CNFs need to be:
...
 
+References:
+- https://opengitops.dev/ - The GitOps Working Group under the CNCF App Delivery SIG.
+- https://www.gitops.tech/ - Collection of information on GitOps by [INNOQ](https://www.innoq.com/en/)
```
Comment (lixuna) [inline doc/whitepaper/Accelerating_Cloud_Native_in_Telco.md]:
```suggestion
- https://www.gitops.tech/ - Collection of information on GitOps by [INNOQ](https://www.innoq.com/en/).
```

**Review by lixuna — CHANGES_REQUESTED** [review]

**Review by lixuna — APPROVED** [review]
lgtm

**Review by jeffsaelens — APPROVED** [review]
LGTM

**Review by electrocucaracha — APPROVED** [review]
/lgtm

**Pull request merged**